templates/todo/projects/nodejs-mongo/.repo/terraform/azure.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: todo-nodejs-mongo-terraform

```

Comment:
Add an entry in `cli/azd/resources/templates.json` for this template to be listed on `azd init` and `azd templates`

Reply:
added